### PR TITLE
fix: tooltip refactor for buyCrypto entry

### PR DESCRIPTION
**Pull request by ChefBingbong**
### Summary
📝📱💰

This pull request enhances the `WalletInfo` component to help users buy more crypto, refactors the `useTooltip` hook to use `lodash` and improve performance, and updates the FAQ content for the `BuyCrypto` view. It also adds new types for the `useTooltip` hook.

> _We're sailing on the crypto sea, me hearties, yo ho ho_
> _We're buying more BNB when our balance is low_
> _We're fixing up the `useTooltip` hook, with `debounce` and `deviceActions`_
> _We're moving all the FAQ items to a separate file, to avoid distractions_

### Walkthrough
*  Add `InfoFilledIcon` component to `WalletInfo` component and adjust layout and logic of `buyCryptoTooltip` component ([link](https://github.com/pancakeswap/pancake-frontend/pull/7266/files?diff=unified&w=0#diff-667fa9ffb5f38ba848d89c70e2b060c60ce67fee0185391f612eb425169911beR15), [link](https://github.com/pancakeswap/pancake-frontend/pull/7266/files?diff=unified&w=0#diff-667fa9ffb5f38ba848d89c70e2b060c60ce67fee0185391f612eb425169911beL71-R86), [link](https://github.com/pancakeswap/pancake-frontend/pull/7266/files?diff=unified&w=0#diff-667fa9ffb5f38ba848d89c70e2b060c60ce67fee0185391f612eb425169911beL175-R182), [link](https://github.com/pancakeswap/pancake-frontend/pull/7266/files?diff=unified&w=0#diff-667fa9ffb5f38ba848d89c70e2b060c60ce67fee0185391f612eb425169911beL183-R192))
* Remove FAQ item from `BuyCrypto` view and move FAQ items to separate file ([link](https://github.com/pancakeswap/pancake-frontend/pull/7266/files?diff=unified&w=0#diff-2999e60a5b9510d1e96d02fe7ecd6283d03a0363972374f5d433eb7ecd52b60fL51-L73))
* Add `debounce` function and `DeviceAction` and `Devices` types to `useTooltip` hook and simplify event handling logic ([link](https://github.com/pancakeswap/pancake-frontend/pull/7266/files?diff=unified&w=0#diff-184e2bdd1dd0d58dab95d4efa5b52f7abd0c62c7833b5dec7f4347f21500d861R24-R33), [link](https://github.com/pancakeswap/pancake-frontend/pull/7266/files?diff=unified&w=0#diff-6fe052fda58ce75c3e945093975d8c35309b36402839fb6166f8cfadde2aa35eL7-R12), [link](https://github.com/pancakeswap/pancake-frontend/pull/7266/files?diff=unified&w=0#diff-6fe052fda58ce75c3e945093975d8c35309b36402839fb6166f8cfadde2aa35eR26-R36), [link](https://github.com/pancakeswap/pancake-frontend/pull/7266/files?diff=unified&w=0#diff-6fe052fda58ce75c3e945093975d8c35309b36402839fb6166f8cfadde2aa35eL52-L53), [link](https://github.com/pancakeswap/pancake-frontend/pull/7266/files?diff=unified&w=0#diff-6fe052fda58ce75c3e945093975d8c35309b36402839fb6166f8cfadde2aa35eL59-R91), [link](https://github.com/pancakeswap/pancake-frontend/pull/7266/files?diff=unified&w=0#diff-6fe052fda58ce75c3e945093975d8c35309b36402839fb6166f8cfadde2aa35eL96-R104), [link](https://github.com/pancakeswap/pancake-frontend/pull/7266/files?diff=unified&w=0#diff-6fe052fda58ce75c3e945093975d8c35309b36402839fb6166f8cfadde2aa35eL110-R111), [link](https://github.com/pancakeswap/pancake-frontend/pull/7266/files?diff=unified&w=0#diff-6fe052fda58ce75c3e945093975d8c35309b36402839fb6166f8cfadde2aa35eL125-R142), [link](https://github.com/pancakeswap/pancake-frontend/pull/7266/files?diff=unified&w=0#diff-6fe052fda58ce75c3e945093975d8c35309b36402839fb6166f8cfadde2aa35eR231))

### CODE REVIEW

File: apps/web/src/components/Menu/UserMenu/WalletInfo.tsx [hunk 1]
```diff
@@ -12,6 +12,7 @@ import {
   useTooltip,
   TooltipText,
   InfoIcon,
+  InfoFilledIcon,
```
Comment (chefjackson) [inline apps/web/src/components/Menu/UserMenu/WalletInfo.tsx:15]: Let's remove unused variable

File: packages/uikit/src/hooks/useTooltip/useTooltip.tsx [hunk 2]
```diff
@@ -237,6 +228,7 @@ const useTooltip = (content: React.ReactNode, options?: TooltipOptions): Tooltip
 
   const portal = getPortalRoot();
   const tooltipInPortal = portal && isInPortal ? createPortal(AnimatedTooltip, portal) : null;
+  // console.log(tooltipInPortal, 'tooltipinportal')
```
Comment (chefjackson) [inline packages/uikit/src/hooks/useTooltip/useTooltip.tsx:231]: Let's remove unused log

File: apps/web/src/components/Menu/UserMenu/WalletInfo.tsx (hunk 1 above)
Comment (ChefBingbong) [reply apps/web/src/components/Menu/UserMenu/WalletInfo.tsx:15]: done

File: packages/uikit/src/hooks/useTooltip/useTooltip.tsx (hunk 2 above)
Comment (ChefBingbong) [reply packages/uikit/src/hooks/useTooltip/useTooltip.tsx:231]: done

**Comment by chefjackson** [comment]
Also pls help update the i18n translations. Checkout the detail [here](https://github.com/pancakeswap/pancake-frontend/actions/runs/5441597143/jobs/9895742405?pr=7266)

**Comment by chefjackson** [comment]
Also pls help resolve the conflict. Thx

**Review by chefjackson — APPROVED** [review]

**Pull request merged**